apps/web/package.json

```diff
@@ -4,13 +4,13 @@
   "private": true,
   "license": "GPL-3.0",
   "scripts": {
-    "dev": "lingui compile; next dev --port 4783",
-    "build": "lingui compile; next build",
+    "dev": "npx lingui compile && next dev --port 4783",
```

Comment:
We always prefer `;` over `&&`

Reply:
but it not run in my env

Reply:
> is `@lingui/cli` in devdependency not working?
to use the lingui need to install lingui global? so if you havent install it , may be run failed

Reply:
and i cant run this project when i pull the latest code